cds-snc/cppd-medical-report
# Add other box for symptoms uknown

# Description

Addresses: https://trello.com/c/UkhzDuBk/372-physicians-can-give-details-on-unknown-symptom-frequency

## Test instructions

- Add a Condition
- Check the Unknown option under "How often do symptoms occur?"
- Notice the Unknown field appears
- Check another box
- Notice the Unknown field disappears
- Check it again and submit to observe the validation message
- Fill it in and submit
